micah-akpan/turing_shopping_cart
# [Feature #168721417] Build paginated products list

#### What does this PR do?
This PR enables users to fetch a paginated list of products

#### Description of Task to be completed?
- Add support for uers to view a list of products 
#### How should this be manually tested?
- Git fetch and git checkout out to `feat/build-products-endpoint`
- Fire server: `yarn dev`
- Make a request using any HTTP REST client to `GET /products?limit=<limit>&page=<page>`
- You should get a successful response
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168721417](https://www.pivotaltracker.com/story/show/168721417)
#### Screenshots (if appropriate)
![Screenshot from 2019-09-25 15-03-49](https://user-images.githubusercontent.com/15838829/65608224-cd642100-dfa5-11e9-8274-069539eb0670.png)

#### Questions:
